client/api/src/notifications.rs

```diff
@@ -139,11 +272,7 @@ impl<Block: BlockT> StorageNotifications<Block> {
 			sinks: Default::default(),
 		}
 	}
-	/// Trigger notification to all listeners.
```

Comment:
Why remove the docs?

Reply:
I simply moved them to the `pub fn` since now this is private; I can restore these if you want though.

Reply:
Yeah, I have seen that later. Is fine that way. SOrry for the noise.